data-raw/sysdata.R

```diff
@@ -0,0 +1,47 @@
+# Code to prepare internal data.
+
+# Unicode block names and ranges from
+# https://en.wikipedia.org/wiki/Unicode_block#List_of_blocks. Consider scraping
+# that to get all of them.
+.unicode_blocks <- data.frame(
```

Comment:
I'd suggest using a tribble here for readability, but I suspect you had a reason for using base data.frame?
If this does eventually get changed to a tribble, maybe put the start/end columns first to improve column positioning.

Reply:
Wanted to avoid the dependency here. I'll set this up for more readability when I make it cleaner. Maybe I'll even use a tribble in suggests just for creation, but save it as a base data.frame.